Analise_horas.py

```diff
@@ -0,0 +1,126 @@
+import pandas as pd
+import datetime
+import numpy as np
+
+
+WORKDAY = datetime.timedelta(hours=9, minutes=0, seconds=0)
+# 8 hours of work +1 lunch time
+WORKDAY_WEEKAND = datetime.timedelta(hours=4)
+# 4 hours of work on weekend
+
+
+def data_anal():
```

Comment:
Sou da opiniao que o Pandas tem uma API terrivel: os nomes dos metodos que eles expoem nao sao nada "readable". With that being said, esta funcao e' um bocado dificil de ler. Era capaz de ser melhor se conseguisses decompor a funcao em varias funcoes com bons nomes, para dar para perceber "de longe" o que e' que isto esta a fazer sem ter que conhecer intimamente a API do Pandas.

Reply:
Como este é capaz de demorar e vou mexer no código a fundo, criei novo branch e vou mexer nesse branch.

Reply:
Estou a iniciar agora SQL, não seria vantajoso para o script incorporar um base dados? Eu seria desnecessário?

Reply:
Acho que o valor academico de incorporar uma base de dados e' enorme! E' um exercicio que pode ser dificil, mas tem boas licoes.

Estou a tentar pensar na utilidade pratica de ter uma base de dados. O programa ja faz o que queres que ele faca. Estas a pensar em persistir informacao numa base de dados depois de criares a tabela, ou para criar a tabela? Acho que vale a pena se tiver utilidade.

Mas seria uma boa oportunidade para experimentares SQL com Python.

Reply:
Pois... acho que é o que vou fazer, nem que seja só pelo exercício. Obrigado